src/backend/src/controllers/teams.controllers.ts

```diff
@@ -127,4 +127,17 @@ export default class TeamsController {
       next(error);
     }
   }
+
+  static async setTeamType(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { teamTypeId } = req.body;
+      const submitter = await getCurrentUser(res);
+
+      const updateTeamType = await TeamsService.setTeamType(submitter, req.params.teamId, teamTypeId);
```

Comment:
can you destructure the teamId just like you do it for the teamTypeId, it just makes it a bit easier to read